### PR TITLE
style: adding header to top of profile page

### PR DESCRIPTION
**Pull request by JackShort**
https://uniswaplabs.atlassian.net/browse/WEB-2285

* this looks like more than it is but the diff is mostly indentation lol

### CODE REVIEW

File: src/nft/components/profile/view/ProfilePage.tsx [hunk 1]
```diff
@@ -33,6 +33,23 @@ const ProfilePageColumn = styled(Column)`
   ${ScreenBreakpointsPaddings}
 `
 
+const ProfileHeader = styled.div`
+  margin-right: 20px;
+  margin-left: 20px;
+  font-size: 28px;
+  font-weight: 500;
+  line-height: 38px;
```
Comment (cbachmeier) [inline src/nft/components/profile/view/ProfilePage.tsx:41]: nit: do we have a common typography we could use here

**Review by cbachmeier — APPROVED** [review]
LGTM minor nit

File: src/nft/components/profile/view/ProfilePage.tsx (hunk 1 above)
Comment (JackShort) [reply src/nft/components/profile/view/ProfilePage.tsx:41]: unfortunately no

Comment (grabbou) [reply src/nft/components/profile/view/ProfilePage.tsx:39]: doesn't theme/components/text work here?

**Pull request merged**